README.rst

```diff
@@ -115,10 +115,11 @@ In order to provide STM32 boards pin configuration using device tree,
 *-pinctrl.dtsi are made available under dts/st directory.
 For each STM32 SoC package a complete and correct -pcintrl.dtsi is available.
 These files are generated using generation scripts available in this repo
-under scripts/genpinctrl. Generation use as input STM32 SoC description database
-delivered with ST STM32CubeMX official tool.
-New set of -pinctrl.dtsi files could be generated following availability of
-new version of STM32CubeMx or change in generation script configuration files.
+under scripts/genpinctrl. Generation use as input the
+`STM32 Open Pin Data <https://github.com/STMicroelectronics/STM32_open_pin_data>`_
+database. New set of -pinctrl.dtsi files could be generated following
+availability of new version of STM32 Open Pin Data or change in generation
+script configuration files.
```

Comment:
Maybe it would be worth adding here what command to run:
`python3 genpinctrl.py [-p /path/to/repo] [-o /path/to/output_dir]`

Reply:
done